NetworkOfPrivateClinics/NetworkOfPrivateClinics/Doctor.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NetworkOfPrivateClinics.CustomExceptions;
+using NetworkOfPrivateClinics.Exceptions;
+
+namespace NetworkOfPrivateClinics
+{
+    public enum DoctorType
+    {
+        Neurologist,
+        Gastroenterologist,
+        Cardiology,
+        Oncologist,
+        Psychiatrist,
+        Dermatologist,
+        Anasthesiologist
+    }
+
+    public class Doctor
+    {
+        public Doctor(int id, string name, string surname, DoctorType type, decimal costOfPermissiom)
+        {
+            DoctorID = id;
+            Name = name;
+            Surname = surname;
+            Type = type;
+            CostOfAdmission = costOfPermissiom;
+            Appointments = new AppointmentFactory("8:00", "17:00");
```

Comment:
Appointment factory should be one layer above this class, and constructor should get appoitments. But it's impossible to create other classes in constructor this way. You violate DI solid principle.
Better create appointment factory at program.cs, get appoitments from factory and pass them to your doctor constructor